internal/params/parameters.go

```diff
@@ -116,6 +116,13 @@ func (p *Parameter) Set() *set {
 	}
 }
 
+func (p *Parameter) Dict() *dict {
```

Comment:
This is user-facing feature - please prepend a line with this feature into CHANGELOG.md